community/contributing/releasing/index.md

```diff
@@ -39,6 +39,13 @@ manager will be the person posting about the releases in [the
 channel](https://app.slack.com/client/T091CRSGH/CHD4ATAMV/). They will evaluate
 the patch, make sure it's appropriate and safe, and merge it if so.
 
+Please make sure your pull request references a bug with enough details to allow
```

Comment:
@plumpy 
By "bug" here, you mean the aberrent behavior, rather than a GH issue, right? 
And, if there's a bug already open, why is it perferred to file a new one?

Reply:
I clarified that I meant GH issue. Let me know if the new wording works or if you think it could be improved further...